### DISCUSSION TITLE
Option to Hide YouTube Shorts Videos from Subscription Feed

### DISCUSSION THREAD

**Discussion by dAtA-TRoN** [libre-tube/LibreTube#4410] [Ideas]
Please add an option to hide YouTube Shorts videos from the subscription feed. Many users, including myself, have found these videos to be less appealing and sometimes even a bit overwhelming. Allowing users to customize their feed in this way would greatly enhance the overall experience.

**Comment by Bnyro** [comment]
That's already possible, there's a filter option in the feed.

**Comment by olegantonyan** [comment]
In that filter you can only select 1 item: All, Videos, Livestreams, Shorts. You cannot select videos and livestreams while leaving shorts off
